### DISCUSSION TITLE
My statblock doesn´t work, what´s wrong with it?

### DISCUSSION THREAD

**Discussion by Spieluin** [Obsidian-TTRPG-Community/fantasy-statblocks#154] [Q&A] [answered]
![Screenshot 2023-03-16 095327](https://user-images.githubusercontent.com/111654542/225597476-2269be1b-0a3f-4bee-ab3d-8721148b9639.png)

**Comment by valentine195** [comment]
It’s probably the `<ability-score>` line.

**Comment by Spieluin** [comment, marked as answer]
Thank you for replying, but it wasn´t that. I got help from Reddit and I
placed some spaces wrong.
